src/PatientChecking/PatientChecking/Views/Shared/_Layout.cshtml

```diff
@@ -1,48 +1,77 @@
 ﻿<!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
 <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - PatientChecking</title>
-    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <title>@ViewData["Title"] | PatientChecking</title>
     <link rel="stylesheet" href="~/css/site.css" />
+    <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
```

Comment:
please add js files as part of the project so the cshtml can refer them as local assets.

Reply:
Js files have been added to project as local assets